src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String DELIMITER =",";
+    public static final String OPERATION ="buy";
+    int supplyAmount = 0;
+    int buyAmount = 0;
+    int result = 0;
+    String operation;
+    String amount;
+    List<String> fileData = null;
+    StringBuilder stringBuilder = new StringBuilder();
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method 'getStatistic' should be divided into smaller methods, each responsible for one task, such as reading data from the file, processing the data, and writing data to the file.